kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -68,10 +68,12 @@ def __init__(self, name, init_config, instances):
             self.instance.get('monitor_all_broker_highwatermarks', False)
         )
         self._consumer_groups = self.instance.get('consumer_groups', {})
-        self._kafka_client = self._create_kafka_admin_client()
-        self._kafka_version = self.instance.get('kafka_client_api_version')
-        if isinstance(self._kafka_version, str):
-            self._kafka_version = tuple(map(int, self._kafka_version.split(".")))
+
+        kafka_version = self.instance.get('kafka_client_api_version')
```

Comment:
I think we should still save this in case we need to create a new client

Reply:
I'd be tempted to push back on this in the name of YAGNI.

Reply:
I'm fundamentally against that principle, but this is so small that yeah, it makes no difference 🙂